.github/workflows/unstable.yml

```diff
@@ -27,6 +28,11 @@ jobs:
           bundler-cache: true
         continue-on-error: ${{ (matrix.ruby-version == 'ruby-head') || (matrix.ruby-version == 'jruby-head') }}
 
-      - run: |
-          bundle exec rake
+      - name: Run tests
+        run: bundle exec rake
+        if: matrix.ruby-version != 'truffleruby-head'
         continue-on-error: ${{ (matrix.ruby-version == 'ruby-head') || (matrix.ruby-version == 'jruby-head') }}
+
+      - name: Run specs (truffleruby)
+        run: bundle exec rake spec
+        if: matrix.ruby-version == 'truffleruby-head'
```

Comment:
This should also continue-on-error right? We don't want the test suite to break

Reply:
This workflow is a daily, so it won't be shown on PRs or even on commits, only at https://github.com/simplecov-ruby/simplecov/actions/workflows/unstable.yml.
So I think there is little value to hide the red icon if it's actually failing.

Reply:
Ah but the other steps have `continue-on-error: ${{ (matrix.ruby-version == 'ruby-head') || (matrix.ruby-version == 'jruby-head') }}`, I see.
I think it's best to remove those, or simplify to `continue-on-error: true` then.

Reply:
probably continue-on-error: true yeah.

I recently took a deep dive on it and yeah it prevents the red from showing up... that saying if it's just daily I think we can display it, the badge in the README shouldn't be affected so maybe you are right and we can remove them.

Reply:
I'll push a commit to remove the `continue-on-error` lines then, it seems clearer that way with no practical downside. Can always be changed if we find a downside to it.